scripts/crunner.py

```diff
@@ -215,9 +207,10 @@ def print_output(*args):
   spitfire.runtime.udn.set_accelerator(
     options.enable_c_accelerator, enable_test_mode=True)
 
-  compiler_args = Compiler.args_from_optparse(options)
-  compiler = Compiler(**compiler_args)
-  
+  compiler_args_from_optparse = (
+      spitfire.compiler.compiler.Compiler.args_from_optparse(options))
+  compiler = spitfire.compiler.compiler.Compiler(**compiler_args_from_optparse)
```

Comment:
(I've condensed that particular example to be on 1 line.)


I know it's overly verbose, but I think it's clearer because there is no
question where the functions/classes are coming from.  Here are the choices:

```python
import spitfire.compiler.compiler
import spitfire.compiler.options
import spitfire.compiler.util
import spitfire.compiler.visitor
import spitfire.runtime
import spitfire.runtime.runner
import spitfire.runtime.udn
# ...
spitfire.compiler.visitor.print_tree(self.compiler._parse_tree)
# ...
spitfire.runtime.import_module_symbol(module_name)
# ...
spitfire.compiler.options.add_common_options(op)
# ...
compiler_args = spitfire.compiler.compiler.Compiler.args(options)
```

vs

```python
from spitfire.compiler import compiler as sptc_compiler
from spitfire.compiler import options as sptc_options
from spitfire.compiler import util as sptc_util
from spitfire.compiler import visitor as sptc_visitor
from spitfire import runtime as sptr
from spitfire.runtime import runner as sptr_runer
from spitfire.runtime import udn as sptr_udn
# ...
sptc_visitor.print_tree(self.compiler._parse_tree)
# ...
sptr.import_module_symbol(module_name)
# ...
sptc_options.add_common_options(op)
# ...
compiler_args = sptc_compiler.Compiler.args(options)
```